<launch>

<node pkg="joy" type="joy_node" name="joy_node">
<param name="dev" type="string" value="/dev/input/js1"/>
</node>


<node
 pkg="rosserial_python"
 type="serial_node.py"
 name="serial_node">

<param name="port" value="/dev/ttyACM0"/>
<param name="baud" value="57600"/>
</node>


<!--joy_twist-->
<node pkg="chjk" type="omni_node" name="omni_node" output="screen">

<param name="r" value="0.5"/>

</node>

<node name="joy_controller" type="joy_controller_node" pkg="joy_controller" >
<param name="MAX_VEL" value="2.0"/>
<param name="MAX_ANGULAR_VEL" value="3.0"/>
<param name="SPD_TRY" value="255"/>
 </node>


<node
 pkg="rosserial_python"
 type="serial_node.py"
 name="serial_node2">

<param name="port" value="/dev/ttyACM1"/>
<param name="baud" value="57600"/>
</node>

<node
 pkg="rosserial_python"
 type="serial_node.py"
 name="serial_node3">

<param name="port" value="/dev/ttyACM2"/>
<param name="baud" value="57600"/>
</node>


<node pkg="pickup_2020" type="pickup_2020.py" name="pick_task_manager" output="screen">

<param name="slide_pwm" value="45.0"/>
<param name="turn_pwm" value="45.0"/>
<param name="lift_pwm" value="255.0"/>

</node>


</launch>

